jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/TFRecordIO.scala

```diff
@@ -0,0 +1,70 @@
+package ml4ir.inference.tensorflow.data
+import java.io.{ByteArrayOutputStream, File, FileOutputStream}
+
+import com.google.common.hash.Hashing
+import com.google.common.io.LittleEndianDataOutputStream
+import org.tensorflow.example.SequenceExample
+
+object TFRecordIO {
+  val kMaskDelta = 0xa282ead8
```

Comment:
A comment on what/why this number is used will be helpful.

Reply:
magic!

Reply:
TBH, I'll just delete this.  There is a much better way of TFRecord IO in scala.  It's called "spark" ;P